lib/shopify_api/logger.rb

```diff
@@ -0,0 +1,54 @@
+# typed: false
+# frozen_string_literal: true
+
+module ShopifyAPI
+  class Logger
```

Comment:
I think we are missing a `private` keyword here to establish a public/private interface. Where do you think a good place to draw that line is?

Reply:
True I think `send_to_logger` and `enabled_for_log_level?` should probably be private. `deprecated` and `context` gets overridden in `shopify_app` so I think that needs to stay public.